dspace/etc/postgres/update-sequences.sql

```diff
@@ -47,38 +43,21 @@
 -- org.dspace.content API which is safe to use concurrently and in multiple
 -- JVMs.  The SQL code below will typically only be required after a direct
 -- SQL data dump from a backup or somesuch.
- 
+
 
 -- There should be one of these calls for every ID sequence defined in
 -- database_schema.sql.
```

Comment:
While you are in here, this comment can likely be removed. It's not as relevant now as we have no database_schema.sql... plus, *most* object IDs are now UUIDs (and don't need a sequence).